holoviews/plotting/plot.py

```diff
@@ -166,7 +166,9 @@ class DimensionedPlot(Plot):
         of plotting. Allows selecting normalization at different levels
         for nested data containers.""")
 
-    projection = param.ObjectSelector(default=None)
+    projection = param.Parameter(default=None, doc="""
```

Comment:
Again, I would like the docstring to say a bit more about what is required to set a custom projection...